docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Common Label and Annotation Recommendations
+---
+
+{% capture overview %}
+Many tools can be used to visualize and manage Kubernetes objects. This goes beyond
+tools such as dashboard or kubectl and includes many tools in the Kubernetes ecosystem.
+To enable interoperability between tools, a common set of labels and annotation
+will allow objects to be described in a common manner that all tools can understand.
+
+In addition to supporting tooling, the common labels and annotations describe
+applications, in a generic sense, in a way that can queried or are otherwise useful
+when operating applications.
+{% endcapture %}
+
+{% capture body %}
+The recommended object metadata is broken apart so that some of it is labels while
+other parts are annotations. Labels are shorter but can be used when querying
+the Kubernetes API. Annotations allow for longer data that's not meant to be
+queried.
+
+The metadata is organized around the concept of an Application. Kubernetes is not
+a Platform as a Service and doesn't have or enforce a formal notion of an application.
+Instead, applications are informal, described with metadata, and the definition of what an
+application contains is loose.
+
+Note, these are recommended labels and annotations. They aid the use of managing
+applications but are not required for any core tooling to work.
+
+One common theme you'll notice about the labels and annotations is that they
+contain a prefix of `app.kubernetes.io`. Labels and annotations without a prefix
+are considered private to users. These are meant to be common labels that do not
+interfere with existing or custom user labels. To accomplish this a prefix is
+used.
+
+## Labels
+
+| Key                                 | Description           | Example  |
+| ----------------------------------- | --------------------- | -------- |
+| app.kubernetes.io/name              | The name of the application stack | wordpress |
```

Comment:
Suggestion - for each example, surround it with the ` character to apply a code style. Not 100% convinced this is correct, but since we’re referencing them as objects it seems relevant per the style guide: https://kubernetes.io/docs/home/contribute/style-guide/#use-code-style-for-object-field-names